docs/guides/security-component-migration.md

```diff
@@ -0,0 +1,819 @@
+| `Form`                             | Vertical margin for form items and fieldsets is no longer included by default, use `<Stack>` for vertical spacing | `@carbon/react`        | Low    |
+| `FormGroup`                        | [Carbon v11 Migration guide][116]                                                                                 | `@carbon/react`        | Low    |
+| `FormLabel`                        | The FormLabel component no longer applies margin                                                                  | `@carbon/react`        | Low    |
+| `GettingStartedLayout`             | Layout modules are under review                                                                                   | —                      | Low    |
+| `Grid`                             | [Carbon v11 Migration guide][106]                                                                                 | `@carbon/react`        | Medium |
+| `Header`                           | **Deprecated:** [View replacement][6]                                                                             | `@carbon/ibm-products` | Medium |
+| `HeaderMenu`                       | Updated from a class to functional component                                                                      | `@carbon/react`        | Low    |
+| `HeaderNavigation`                 | Updated from a class to functional component                                                                      | `@carbon/react`        | Low    |
+| `ICA`                              | **Renamed** - See [Big Numbers](#bignumbers)                                                                      | `@carbon/ibm-products` | Low    |
+| `Icon`                             | **Removed** [View replacement][16]                                                                                | `@carbon/react`        | Medium |
+| `IconButton`                       | **Deprecated:** [View replacement][7]                                                                             | `@carbon/react`        | High   |
+| `IconButtonBar`                    | **Deprecated:** [View replacement][8]                                                                             | `@carbon/ibm-products` | High   |
+| `InlineLoading`                    | The deprecated success prop has been deprecated. Please use status="finished" instead                             | `@carbon/react`        | Low    |
+| `InlineNotification`               | [Carbon v11 Migration guide][117]                                                                                 | `@carbon/react`        | Medium |
+| `InteractiveTag`                   | **Removed**                                                                                                       | —                      | High   |
```

Comment:
I don’t recall seeing InteractiveTag in the old v1 package? Maybe it had another name, but wanted to compare since Carbon core has some new unstable/experimental [`InteractiveTag`s](https://react.carbondesignsystem.com/?path=/docs/experimental-unstable-interactivetag--overview)